tests/test_views.py

```diff
@@ -52,12 +54,18 @@ def test_request_route(self):
         assert f"api/route/{response.data.get('id')}" in response.data.get("status-url")
         assert isinstance(uuid.UUID(response.data.get("id")), uuid.UUID)
 
-    
+        # Test that the correct polarrouteserver version number is returned
+        # in the response
+        assert response.data.get("polarrouteserver-version") == polarrouteserver_version
+
         # Test that requesting the same route doesn't start a new job.
         # request the same route parameters
         request = self.factory.post("/api/route/", data=data, format="json")
         response2 = RouteView.as_view()(request)
         assert response.data.get('id') == response2.data.get('id')
+        assert response.data.get("polarrouteserver-version") == response2.data.get(
```

Comment:
I'm not sure that we need to check this again here, unless I'm missing something.